src/Database/Expression/ValuesExpression.php

```diff
@@ -187,10 +187,10 @@ public function getValues(): array
      * Sets the query object to be used as the values expression to be evaluated
      * to insert records in the table.
      *
-     * @param \Cake\Database\Query $query The query to set
+     * @param \Cake\Database\QueryInterface $query The query to set
      * @return $this
      */
-    public function setQuery(Query $query)
+    public function setQueryInterface(QueryInterface $query)
```

Comment:
`setQueryInterface` and `getQueryInterface` look like a "find and replace" mistake?

Reply:
Yup find/replace fail :).